autopilot/ap-batch-transform.ipynb

```diff
@@ -0,0 +1,1297 @@
+{
```

Comment:
<p>Not needed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4312/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>